environment-dev.yml

```diff
@@ -30,8 +30,8 @@ dependencies:
   - shapely >=2.0
   - sparse
   - toolz
-  - xarray >=2023.11.0, !=2024.6.0
-  - xclim >=0.53.2, <0.54
+  - xarray >=2023.11.0, !=2024.6.0, <2024.10.0 #FIXME: 2024.10.0 breaks rechunker with zarr, https://github.com/pangeo-data/rechunker/issues/154
+  - xclim >=0.54, <0.55
```

Comment:
@Zeitsperre Are we better to always only support 1 version, or could we have `xclim >=0.53.2, <0.55` for more flexibility?

Reply:
We've made a fair amount of breaking changes in recently in `xclim` in order to ensure that `xscen` works properly (which is great! Fewer bugs in `xclim`). If we want to test across more versions, we should reconfigure a CI build to run against the oldest supported version of `xclim`.

Reply:
Alright. This is not the PR for it, but I think that having a way to actually test the oldest supported version of `xclim` would be useful (including for packages further down the line like xHydro).